src/index.scss

```diff
@@ -89,4 +89,18 @@ button {
     border: 0;
     padding: 0;
   }
-}
\ No newline at end of file
+}
+
+/* Sticky table headers */
+thead th {
+    position: sticky;
+    top: 0;
+    background-color: #fff;
+}
+/* No gap at top of list to prevent list bleedthrough above thead */
+.js-enabled .govuk-tabs__panel {
```

Comment:
We should remove the `.js-enabled` selector so that this still works when javascript is disabled.

Reply:
The top padding this is overriding is only present when JS is enabled - it's the `padding: 30px 20px;` in `.js-enabled .govuk-tabs__panel` in _spacing.scss (it's within a media query there so presumably should be in one here too, sorry for missing that).